Add back button to appbar and button section

korneliakobiela opened the pull request:
[Issue] #1188
[Problem] [UiComponents] App doesn't have back button in every place.
[Solution] - Add backbutton to sections Appbar and Button

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

korneliakobiela committed:
Add backbutton to appbar and button section

[Issue] #1188
[Problem] [UiComponents] App doesn't have back button in every place.
[Solution] - Add backbutton to sections Appbar and Button

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

HunseopJeong reviewed (approved):
LGTM

lslachciak commented on examples/mobile/UIComponents/components/buttons/contained/index.html:
```diff
@@ -11,6 +11,9 @@
 	<div class="ui-page" id="ContainedButtonsDemo">
+		<div class="ui-appbar-left-icons-container">
+			<a href="#" class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></a>
+		</div>
```
please double check code here, view is broken in this example on my PC
![image](https://user-images.githubusercontent.com/2246331/87505524-5baab000-c669-11ea-8204-14c9702241a1.png)

lslachciak commented on examples/mobile/UIComponents/components/appbar/multiline/multiline-title.html:
```diff
@@ -12,6 +12,9 @@
 <body>
 	<div class="ui-page" id="multilineTitle">
 		<header>
+			<div class="ui-appbar-left-icons-container">
```
we need to decide and be consistent if we add back button to submenus or examples too.
This is example not submenu.
If we add back button to examples we will need to add to all of them
e.g. also
examples/mobile/UIComponents/components/etc/units/units.html
examples/mobile/UIComponents/components/etc/units/fonts.html
examples/mobile/UIComponents/components/etc/interactive/coverflow/index.html
examples/mobile/UIComponents/components/etc/interactive/interactive3D/index.html
examples/mobile/UIComponents/components/etc/about/index.html

lslachciak commented on examples/mobile/UIComponents/components/appbar/multiline/subtitle.html:
```diff
@@ -12,6 +12,9 @@
 <body>
 	<div class="ui-page" id="subtitle">
 		<header>
```
ditto.

lslachciak commented on examples/mobile/UIComponents/components/buttons/contained/body-area/2-buttons.html:
```diff
@@ -11,6 +11,12 @@
 
 <body>
 	<div class="ui-page" id="ContainedButtonsBodyArea2Buttons">
+		<header>
```
ditto.

lslachciak reviewed (changes requested):
Please check my comments in code,
also I see missing back button in following submenus
examples/mobile/UIComponents/components/toast/index.html
examples/mobile/UIComponents/components/textfields/index.html
examples/mobile/UIComponents/components/etc/index.html
examples/mobile/UIComponents/components/etc/interactive/index.html
examples/mobile/UIComponents/components/etc/graphs/index.html

korneliakobiela committed:
Add back button to half etc elements

[Issue] #1188
[Problem] - No backbutton in some places
[Solution] - Add backbutton

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

lslachciak reviewed (approved):
seems good.
Toast, Checkbox and Etc main menu will need back button in other PR
